templates/web/common/includes/donate_banner.tt.html

```diff
@@ -24,11 +24,14 @@
 <script>
   let d = new Date();
   let bannerID = document.getElementById('donate-banner');
+  let getDomain = window.location.origin.split('.');
 
   function setBannerCookie(bcname, bcval, bcexdays) {
     d.setTime(d.getTime() + (bcexdays*60*60*24*1000));
     let expires = 'expires=' + d.toUTCString();
-    document.cookie = bcname + '=' + bcval + ';' + expires + ';path=/';
+    // Apply cookie for every type of link contains '.openfoodfacts.'
+    let domain = 'domain=.openfoodfacts.' + getDomain[2];
```

Comment:
Product opener is not only used by openfoodfacts but also openproductsfacts, etc. 
We should never hardcode domain.

```suggestion
    // Apply cookie for whole primary domain (xxx.org)
    let domain = 'domain=.' + getDomain.slice(-2).join(".");
```

Reply:
Added this change in `204edea` commit.

Reply:
Didn't worked, I will figure it out soon.

Reply:
I used `.slice(1)` to delete `http://world`. `.slice(-2)` caused that close button didn't worked.